Solutions/MimecastTTP/SolutionMetadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "mimecast",
+    "offerId": "mimecastttp",

```

Comment:
Please update the offer ID as per this document 
https://github.com/Azure/Azure-Sentinel/tree/master/Solutions#author-and-publish-a-solution-offer